ui/constants/pages.js

```diff
@@ -34,6 +34,7 @@ exports.HELP = 'help';
 exports.LIBRARY = 'library';
 exports.LISTS = 'lists';
 exports.PLAYLISTS = 'playlists';
+exports.HISTORY = 'history';
```

Comment:
`WATCH_HISTORY` will be better, as we will have "Search History" coming up as well.